bee-signing/tests/ternary_seed.rs

```diff
@@ -1,7 +1,9 @@
 // Copyright 2020-2021 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-use bee_crypto::ternary::sponge::{Kerl, Sponge};
+#[allow(deprecated)]
+use bee_crypto::ternary::sponge::Kerl;
+use bee_crypto::ternary::sponge::Sponge;
```

Comment:
I think I would even deprecate Sponge tbh... The Curl version in iota-crypto doesn't implement this trait.